VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts

```diff
@@ -251,14 +251,14 @@ export class ParticipantsPanelComponent implements OnInit, AfterViewInit, OnDest
         this.videoCallService.lowerAllHands(this.conferenceId);
     }
 
-    lowerParticipantHand(participantId: string) {
-        const participant = this.participants.find(x => x.id === participantId);
+    lowerParticipantHand(participant: PanelModel) {
+        const p = this.participants.find(x => x.id === participant.id);
```

Comment:
passing in participant with everything we need on it.. why find?

Reply:
the pexip id is undefined and unknown until the hearing starts and can change between connections